packages/ibm-products-styles/src/components/ConditionBuilder/styles/_conditionBuilderCondition.scss

```diff
@@ -44,3 +56,15 @@ $block-class: #{c4p-settings.$pkg-prefix}--condition-builder;
   text-overflow: ellipsis;
   text-wrap: nowrap;
 }
+.#{$block-class}__group-preview {
+  opacity: 0.5;
+  pointer-events: none;
+}
+.#{$block-class}__connector-disabled {
+  display: flex;
+  min-width: 4rem;
```

Comment:
Is it possible to replace it with a Carbon token?

Reply:
resolved as advised